GVRf/Framework/framework/src/main/java/org/gearvrf/GVRContext.java

```diff
@@ -524,9 +537,23 @@ public GVRSceneObject getAssimpModel(String assetRelativeFilename)
      */
     public GVRSceneObject getAssimpModel(String assetRelativeFilename,
             EnumSet<GVRImportSettings> settings) throws IOException {
-        return GVRImporter.getAssimpModel(this, assetRelativeFilename, settings);
+        return getAssimpModel(assetRelativeFilename, settings, null);
```

Comment:
passing a null argument to IAssetEvents would result in an NPE inside the function call.